### PR TITLE
Feature/user auth

### PR DESCRIPTION
**Pull request by SlWa99**

### CODE REVIEW

**Review by emilie-h — CHANGES_REQUESTED** [review]

File: db/db_create.sql [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+DROP TABLE IF EXISTS `serie`;
+DROP TABLE IF EXISTS `publisher`;
+DROP TABLE IF EXISTS `bookshelf`;
+
+--
+-- Table structure for table `author`
+--
+CREATE TABLE `author` (
+    `id_author` INT UNSIGNED NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(45) NOT NULL,
```
[the remark this answers is not in the document]
Comment (SlWa99) [reply db/db_create.sql:19]: @Kalysso Can you explain this choice ?

File: db/db_create.sql [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    UNIQUE KEY `id_author_UNIQUE` (`id_author`),
+    UNIQUE KEY `name_UNIQUE` (`name`)
+);
+
+--
+-- Table structure for table `bookshelf`
+--
+CREATE TABLE `bookshelf` (
+    `id_bookshelf` INT UNSIGNED NOT NULL AUTO_INCREMENT,
+    `name` VARCHAR(45) NOT NULL,
```
[the remark this answers is not in the document]
Comment (SlWa99) [reply db/db_create.sql:31]: @Kalysso Same here

File: db/db_create.sql [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+        REFERENCES `work` (`id_work`)
+);
+
+--
+-- Table structure for table `review`
+--
+CREATE TABLE `review` (
+    `id_review` INT UNSIGNED NOT NULL AUTO_INCREMENT,
+    `ratings` TINYINT DEFAULT NULL,
+    `favorite` TINYINT UNSIGNED DEFAULT NULL,
```
[the remark this answers is not in the document]
Comment (SlWa99) [reply db/db_create.sql:114]: @Kalysso Same here

[the remark this answers is not in the document]
Comment (Kalysso) [reply db/db_create.sql:114]: BOOL is TINYINT(1) for MySQL. I'll change for better reading

**Review by emilie-h — APPROVED** [review]

**Review by emilie-h — DISMISSED** [review]

**Review by emilie-h — DISMISSED** [review]

**Review by BetterBlood — APPROVED** [review]

**Pull request merged**